CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 24.10+ (???)
 ------------------------------------------------------------------------
 - Fix: [#2676] Cargo labels are misaligned in vehicle window.
+- Feature: [#2695] Ability to change the music sort order of the Options - Music and Edit Music Selection windows.
```

Comment:
In the changelog we list Features before Fixes. I know this isn't obvious, sorry!